beacon-chain/sync/subscriber_beacon_blocks.go

```diff
@@ -31,15 +32,16 @@ func (r *Service) beaconBlockSubscriber(ctx context.Context, msg proto.Message)
 	if err != nil {
 		return err
 	}
-
-	// Broadcast the block on a feed to notify other services in the beacon node
-	// of a received block (even if it does not process correctly through a state transition).
-	r.blockNotifier.BlockFeed().Send(&feed.Event{
-		Type: blockfeed.ReceivedBlock,
-		Data: &blockfeed.ReceivedBlockData{
-			SignedBlock: signed,
-		},
-	})
+	if !featureconfig.Get().SlasherP2P {
```

Comment:
Why is it being gated here if it wasn't earlier?

Reply:
i send all the blocks to block stream here already in the new code 
https://github.com/prysmaticlabs/prysm/blob/be7b669737cf8838fbb02c0b2ad66ec6a3994648/beacon-chain/sync/validate_beacon_blocks.go#L83-L102

Reply:
i dont want to send them twice